portfolio/src/main/webapp/index.html

```diff
@@ -22,30 +53,36 @@ <h2>Introduction</h2>
       <br>
       <p>Many different aspects of Computer Science have made me fall in love with the subject,
         specifically Machine Learning/Deep Learning, Web Development, and Software Engineering.
-        I also am part of an organization, WECARE, that advocates for increased STEM education
+        I also help lead an organization, WECARE, that advocates for increased STEM education
         for women all around the globe, a topic I'm incredibly passionate about. 
       </p>
       <br>
       <p>
         In my free time, I enjoy going to the gym, watching some TV, and cheering on
-        some Golden State Warriors Basketball. 
+        some Golden State Warriors Basketball. (NBA Finals prediction: Dubs in 6 - June 7, 2022)
```

Comment:
> Dubs in 6 

What does this mean?

Reply:
I'll clarify this on my portfolio, but I was basically saying Warriors would win the NBA finals in 6 games.

Reply:
Doesn't necessarily need clarification, I just don't know anything about basketball lol